<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesUnits</class>
 <widget class="QWidget" name="PreferencesUnits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_units">
     <property name="title">
      <string>Unit system</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>System</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="metric">
        <property name="text">
         <string>&amp;Metric</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="imperial">
        <property name="text">
         <string>Imperial</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="personalize">
        <property name="text">
         <string>Personali&amp;ze</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="units_group">
     <property name="title">
      <string>Individual settings</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Depth</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="meter">
        <property name="text">
         <string>meter</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="feet">
        <property name="text">
         <string>feet</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Pressure</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="bar">
        <property name="text">
         <string>bar</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="psi">
        <property name="text">
         <string>psi</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Volume</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="liter">
        <property name="text">
         <string>&amp;liter</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="cuft">
        <property name="text">
         <string>cu ft</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Temperature</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="celsius">
        <property name="text">
         <string>celsius</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QRadioButton" name="fahrenheit">
        <property name="text">
         <string>fahrenheit</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Weight</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="kg">
        <property name="text">
         <string>kg</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QRadioButton" name="lbs">
        <property name="text">
         <string>lbs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Time units</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label13">
        <property name="text">
         <string>Ascent/descent speed denominator</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="vertical_speed_minutes">
        <property name="text">
         <string>Minutes</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="vertical_speed_seconds">
        <property name="text">
         <string>Seconds</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="title">
      <string>GPS coordinates</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>Location Display</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="gpsTraditional">
        <property name="text">
         <string>traditional (dms)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="gpsDecimal">
        <property name="text">
         <string>decimal</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
